src/LaunchDarkly.Client/LdClient.cs

```diff
@@ -258,8 +258,8 @@ protected virtual void Dispose(bool disposing)
         {
             Logger.LogInformation("Closing LaunchDarkly client.");
             //We do not have native resource, so the boolean parameter can be ignored.
```

Comment:
Just an FYI, this isn't actually how the .NET dispose pattern works.  Dispose calls this method with Dispose(true), so you always check the ```disposing``` variable for managed variables.  If something inherits this class, which uses native resources, then they would add the deconstructor which passes false, at which point, you don't double dispose.

Reply:
Hmm, yes, this does look a little odd. Even though we don't encourage subclassing LdClient, it'd be good to follow the standard pattern.

Reply:
I definitely agree people shouldn't be extending this.  Sealing the class would be my first recommendation.